app/models/concerns/pluckable.rb

```diff
@@ -16,8 +16,9 @@ def self.pluck_columns
       columns = table_columns + ref_columns
       columns.each do |column|
         relation = column.to_s.split('.')[0].pluralize if column.to_s.include? '.'
-        next unless relation and relation != self.table_name
+        next unless relation && relation != self.table_name
```

Comment:
Yes, most of the time && is a better choice (these are slightly different operators) but if you prefer and and it works, it's ok with me.